region/maridia/outer.json

```diff
@@ -2058,6 +1618,26 @@
               "unlock": null
```

Comment:
Not sure how to comment outside of the changed portions so doing it here instead, probably easiest if you do the change. Around row 1594 there's a Mama Turtles req for canSwim or SuitlessMaridia that tries to combine that with reqs that can't be done suitless such as canFly.